Report builder: fails to build

```
   Compiling service v0.1.0 (/Users/m2/GitHub/open-msupply/server/service)
error[E0433]: failed to resolve: could not find `select` in `tokio`
   --> service/src/sync/synchroniser.rs:144:16
    |
144 |         tokio::select! {
    |                ^^^^^^ could not find `select` in `tokio`

warning: unused import: `time::Duration`
 --> service/src/sync/synchroniser.rs:7:22
  |
7 | use std::{sync::Arc, time::Duration};
  |                      ^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0433`.
warning: `service` (lib) generated 1 warning
error: could not compile `service` due to previous error; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/server/android/Cargo.toml
+++ b/server/android/Cargo.toml
@@ -11,12 +11,14 @@ name = "remote_server_android"
 [dependencies]
 repository = { path = "../repository" }
 service = { path = "../service" }
-server = { path = "../server", default-features = false, features= [ "android" ] }
+server = { path = "../server", default-features = false, features = [
+    "android",
+] }
 
-actix-web = { version= "4.0.1", features = ["rustls"] } 
+actix-web = { version = "4.0.1", features = ["rustls"] }
 android_logger = "0.11.0"
-jni = { version = "0.19.0"}
+jni = { version = "0.19.0" }
 log = "0.4.14"
 once_cell = "1.9.0"
 rcgen = "0.9.2"
-tokio = { version = "1.17.0", features = ["macros" ] }
+tokio = { version = "1.17.0", features = ["macros"] }

--- a/server/repository/Cargo.toml
+++ b/server/repository/Cargo.toml
@@ -11,24 +11,28 @@ path = "src/lib.rs"
 util = { path = "../util" }
 
 chrono = { version = "0.4", features = ["serde"] }
-diesel = { version = "1.4.7", default-features = false, features = ["r2d2", "numeric", "chrono", "32-column-tables"] }
+diesel = { version = "1.4.7", default-features = false, features = [
+    "r2d2",
+    "numeric",
+    "chrono",
+    "32-column-tables",
+] }
 diesel-derive-enum = { version = "1.1.1", default-features = false }
 diesel_migrations = "1.4.0"
 futures-util = "0.3.15"
 libsqlite3-sys = { version = "0.22.2", features = ["bundled"], optional = true }
 uuid = { version = "0.8", features = ["v4"] }
-serde = {version = "1.0.126", features = ["derive"]}
+serde = { version = "1.0.126", features = ["derive"] }
 serde_json = "1.0.66"
 thiserror = "1"
 log = "0.4.14"
 
 [dev-dependencies]
-tokio = {version = "1.17.0", features=["time","rt-multi-thread","macros"]}
+tokio = { version = "1.17.0", features = ["time", "rt-multi-thread", "macros"] }
 actix-rt = "2.6.0"
 
 [features]
 default = ["sqlite"]
 sqlite = ["diesel/sqlite", "libsqlite3-sys", "diesel-derive-enum/sqlite"]
 memory = ["diesel/sqlite", "libsqlite3-sys", "diesel-derive-enum/sqlite"]
-postgres = ["diesel/postgres" , "diesel-derive-enum/postgres"]
-
+postgres = ["diesel/postgres", "diesel-derive-enum/postgres"]

--- a/server/service/Cargo.toml
+++ b/server/service/Cargo.toml
@@ -22,9 +22,9 @@ serde = "1.0.126"
 serde_json = "1.0.66"
 serde_yaml = "0.8.24"
 tera = "1"
-tokio = "1.17.0"
+tokio = { version = "1.17.0", features = ["macros", "sync", "time"] }
 headless_chrome = "0.9"
-failure = "0.1.8" 
+failure = "0.1.8"
 
 [dev-dependencies]
 actix-rt = "2.6.0"

